CRM/Case/Task.php

```diff
@@ -86,6 +86,11 @@ public static function &tasks() {
           'class' => 'CRM_Case_Form_Task_Restore',
           'result' => FALSE,
         ),
+        5 => array(
+          'title' => ts('Print/Merge Document'),
```

Comment:
Keeping with #8439 this string should be `'Print/merge document'`. (case-sensitive)